### ISSUE TITLE
Chrome screen reader unable to read content of Downshift menu

### ISSUE THREAD

**Issue by SteelFace87** [downshift-js/downshift#1140]
- `downshift` version: 6.0.2
- `node` version: 12.16.2
- `npm ` version: 6.14.4
- `chrome ` 84.0.4147.105 (Official Build) (64-bit)
- `macOS` Catalina 10.15.5 

**Relevant code or config (Create React App)**
```
import React from "react";
import Downshift from "downshift";

export const DownshiftExample = () => {
  const items = [
    { value: "value1", id: "id1" },
    { value: "value2", id: "id2" },
    { value: "value3", id: "id3" },
  ];

  const itemToString = (item) => (item.value ? item.value : "");
  return (
    <div>
      <h1>Downshift Example</h1>
      <div>
        <Downshift itemToString={itemToString}>
          {({
            getLabelProps,
            getInputProps,
            getMenuProps,
            getItemProps,
            getToggleButtonProps,
            isOpen,
            highlightedIndex,
          }) => (
            <div>
              <label {...getLabelProps()}> Test Screen Reader</label>

              <input
                readOnly={true}
                placeholder="select"
                {...getInputProps()}
              ></input>
              <button {...getToggleButtonProps()}>
                {isOpen ? "Close Menu" : "Open Menu"}
              </button>

              <ul {...getMenuProps()}>
                {isOpen
                  ? items.map((item, i) => (
                      <li
                        key={i}
                        {...getItemProps({
                          item,
                          key: item.id,
                          style: {
                            backgroundColor:
                              highlightedIndex === i ? "green" : "blue",
                          },
                        })}
                      >
                        {item.value}
                      </li>
                    ))
                  : null}
              </ul>
            </div>
          )}
        </Downshift>
      </div>
    </div>
  );
};
```

**What you did**:
1) Made a Create React App
2) Installed downshift (npm i downshift)
3) Created a Downshift component, and render in App.js
4) NPM run start

**What happened**:
Chrome does not support VoiceOver for reading contents of a Downshift dropdown menu.  Safari does not have this issue.  More specifically when using the down key when focus is on the input element the menu opens up and screen reader functions as expected.  If the user is to use the toggleButton to open the menu, then screen reader is not able to read content of the dropdown menu.

**Reproduction repository**:
https://github.com/SteelFace87/Downshift-Screenreader-Issue

**Problem description**:
Same as What happened

**Suggested solution**:
Only workaround I have found is to create an aria-live div with opacity:0 and position:absolute to hide the content of the aria live div.  Then using a debounced function updating the content of the aria-live div once the user has stopped scrolling through the dropdown menu.  This however causes issues when using Safari since safari can read the content of the Downshift menu, and on top of that also reads the content of the aria-live div.    The only work around this is to remove the aria tags added by Downshift so that Safari won't read the menu items twice, however this defeats the purpose of using Downshift in the first place.

```
import debounce from 'lodash.debounce';

const [ariaLiveContent, setAriaLiveContent] = useState('');

const updateAriaLive = (content: string) => {
    setAriaLiveContent(content);
  };

const updateAriaLiveDebounced = debounce(
    (content: string) => updateAriaLive(content),
    500,
    { leading: false, trailing: true }
  );

const onDownshiftStateChange = (state: StateChangeOptions<any>) => {
    if (typeof state.highlightedIndex === 'number') {
      updateAriaLiveDebounced(
        `${ariaLabel} ${options[state.highlightedIndex].text}`
      );
    }
  };
 
<div style={{opacity:"0", position:"absolute"}} role="log" aria-live="polite">
            {ariaLiveContent}
</div>
```

**Comment by silviuaavram** [comment]
Hi! Before digging any deeper into this and seeing the example is a standard one, can you repro the problem on either https://www.downshift-js.com/use-combobox#basic-usage or https://www.downshift-js.com/downshift#usage-with-getrootprops ?

**Comment by SteelFace87** [comment]
> Hi! Before digging any deeper into this and seeing the example is a standard one, can you repro the problem on either https://www.downshift-js.com/use-combobox#basic-usage or https://www.downshift-js.com/downshift#usage-with-getrootprops ?

I was able to copy/paste the example from https://www.downshift-js.com/use-combobox#basic-usage into my Create React App and it works with Chrome screen reader.   I have also updated my Github repo so that it renders both the ComboBox version and the original that doesn't work with Chrome screen reader.

**Comment by SteelFace87** [comment]
I did a little further research and it appears that on the working example ( https://www.downshift-js.com/use-combobox#basic-usage ), the focus shifts from the toggleButton to the Input field and this somehow enables the screen reader to perform as expected.   However in the example ( https://www.downshift-js.com/downshift#usage-with-getrootprops ) the focus does not shift over to the input element after clicking the toggleButton and the screen reader is not able to read the menu.  It seems possible that the way focus is handled is playing a role in this bug.

**Comment by silviuaavram** [comment]
It's the reason I will try to emphasise in the documentation to use the hooks. On one hand it might make sense to change Downshift in order to fix things. On another hand I really do not want to add further changes to the component, and focus on the hooks. Can you move to `useCombobox` in your usage?

**Comment by SteelFace87** [comment]
Yes I have already started going the useCombobox route.
